modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -24,8 +24,8 @@ InputParameters validParams<RecomputeRadialReturnIsotropicPlasticity>()
 }
 
 
-RecomputeRadialReturnIsotropicPlasticity::RecomputeRadialReturnIsotropicPlasticity(const InputParameters & parameters) :
-    RecomputeRadialReturn(parameters),
+IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputParameters & parameters) :
+    RadialReturnStressUpdate(parameters),
     _yield_stress_function(isParamValid("yield_stress_function") ? &getFunction("yield_stress_function") : NULL),
     _yield_stress(getParam<Real>("yield_stress")),
```

Comment:
I agree, @sapitts.  Although I agree with @dschwen 's suggestion, this doesn't really have anything to do with the changes in review here. We have multiple development efforts waiting on this PR, and need to get this in without expanding its scope.